core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/sort/SortOpDesc.scala

```diff
@@ -0,0 +1,51 @@
+package edu.uci.ics.texera.workflow.operators.sort
+
+import com.fasterxml.jackson.annotation.{JsonProperty, JsonPropertyDescription}
+import edu.uci.ics.amber.engine.common.workflow.{InputPort, OutputPort}
+import edu.uci.ics.texera.workflow.common.metadata.{OperatorGroupConstants, OperatorInfo}
+import edu.uci.ics.texera.workflow.common.operators.PythonOperatorDescriptor
+import edu.uci.ics.texera.workflow.common.tuple.schema.Schema
+
+class SortOpDesc extends PythonOperatorDescriptor {
```

Comment:
- does this operator run on a single worker? we cannot let it be parallel.
- isn't this operator blocking?